Backends/RasterImage/output-to-image.lisp

```diff
@@ -50,11 +50,10 @@
                 ,@body)
               (,exit-fn (sheet stream)
                 (declare (ignore stream))
-                (image-mirror-image (sheet-mirror sheet)))
+                (image-mirror-image sheet))
               (,enter-fn (sheet stream)
                 (declare (ignore stream))
-                (when ,image
-                  (setf (image-mirror-image sheet) ,image))))
+                nil))
```

Comment:
This makes `image` unused. Does the interface still make sense like that?

Reply:
Related warnings in `drawing-tests.lisp`: `In DEFINE-DRAWING-TEST "RGB Design": The variable #:IMAGE15 is defined but never used.`

Reply:
this interface was broken before the PR< because image-mirror-image is a reader, not the accessor - so there was no function (setf image-mirror-image). This macro is internal and was not used with the argument "in the wild" - internal uses put there always nil, that's why we had no errors. I'm making the argument `image` ignored for now.